app/mailers/request_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class RequestMailer < ApplicationMailer
+  def request_mail request, type
+    @user = request.user
+    @request = request
+    @type = type
```

Comment:
tìm cách truyền biến local vào mail em nhá
https://stackoverflow.com/questions/12317000/actionmailer-pass-local-variables-to-the-erb-template